jdaviz/configs/default/plugins/data_tools/data_tools.py

```diff
@@ -11,22 +12,42 @@
 @tool_registry('g-data-tools')
 class DataTools(TemplateMixin):
     template = load_template("data_tools.vue", __file__).tag(sync=True)
-    file_path = Unicode("").tag(sync=True)
     dialog = Bool(False).tag(sync=True)
     valid_path = Bool(True).tag(sync=True)
     error_message = Unicode().tag(sync=True)
 
-    @observe("file_path")
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        if 'JDAVIZ_START_DIR' in os.environ:
+            start_path = os.environ['JDAVIZ_START_DIR']
+        else:
+            start_path = os.path.abspath(os.path.curdir)
```

Comment:
Could be simplified? Though if you really want, you can also check if the env var is pointing to a real dir, but I am not sure if we have to.

```suggestion
        start_path = os.environ.get('JDAVIZ_START_DIR', os.path.curdir)
```

Reply:
Thanks!